apps/docs/src/common/pages/get-started/project.tsx

```diff
@@ -0,0 +1,115 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
+
+
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <div className="govuk-grid-column-one-third">
+      <aside>
+        <h2>Get started</h2>
+          <ul className="plain">
+            <li><A href="/get-started/prototype">Prototyping for designers</A></li>
+            <li><A href="/get-started/project">Starting a project for developers</A></li>
+          </ul>
+      </aside>
+    </div>
+
+    <div className="govuk-grid-column-two-thirds">
+      <h1>
+        <span className="caption">Get started</span>
+        Starting a project for developers
+      </h1>
+        <p>This guide explains how to set up a project in production using the
+        coded React examples from the Home Office Design System.</p>
```

Comment:
> This guide will show you how to quickly bootstrap a new project that uses the Home Office Design System.
>
> For our purposes, a _project_ is a _monorepo_ (see: Design Decisions) that can contain multiple applications as well as shared libraries and UI components. A project also provides a convenient way to document and share any components that your team builds.

Reply:
This has been amended. What is the link for Design Decisions?